### ISSUE TITLE
remove project.ini_handler

### ISSUE THREAD

**Issue by untzag** [wright-group/yaqc-cmds#265]
requires following issue(s) to be addressed
- [x] #266 
- [ ] #268

**Comment by ksunden** [comment]
Full list of places that use ini handler:

```
$ rg ini_handler
somatic/queue.py
357:        self.ini = wt.kit.INI(self.ini_path)  # I don't use ini_handler here

somatic/modules/zero_tune.py
22:import pycmds.project.ini_handler as ini_handler
25:ini = ini_handler.Ini(os.path.join(main_dir, "somatic", "modules", "zero_tune.ini"))

project/project_globals.py
22:import pycmds.project.ini_handler as ini

devices/InGaAs_array/InGaAs.py
19:from project.ini_handler import Ini

hardware/opas/TOPAS/TOPAS.py
12:from project.ini_handler import Ini
24:        self.ini = project.ini_handler.Ini(__here__ / "TOPAS.ini")

autonomic/coset.py
15:import pycmds.project.ini_handler as ini_handler
31:ini = ini_handler.Ini(ini_path)

devices/PCI-6251/PCI-6251.py
23:import pycmds.project.ini_handler as ini
24:from project.ini_handler import Ini
```

3 are gone via yaq daemons (NI, InGaAs, TOPAS)

#268 doesn't actually use it, using wt.kit.INI instead, but still should transition to toml, I think (one true config format... that or bluesky eats it up)

Zero tune should use config like other modules (not sure why this was missed)

project_globals, it is imported but not used, but removing the import breaks things (though I think moving the import to `__init__` would likely solve it, but the plan is to delete it entirely anyway.)

**Comment by ksunden** [comment]
Further, as far as I can tell, the only code that uses the automatic ini behavior of PycmdsObject  (outside of OPA, NI, InGaAs) is that zero tune module, so as soon as that is updated, qtypes may be more viable

**Comment by untzag** [comment]
@ksunden are you sure it's not used in coset.py :thinking:

**Comment by ksunden** [comment]
@untzag ini_handler is used in coset.py, but _not_ with the auto saving features of PycmdsObject, the only things from project.classes used in coset.py are Combo and Bool, each instantiated with no parameters

**Comment by ksunden** [comment]
Including changes from #289, #291, and #292 (not yet merged, admittedly)

Only the Coset usage is left (well, and the unused import in project_globals)

**Comment by ksunden** [comment]
closed by #333
